layers/buffer_validation.cpp

```diff
@@ -5611,6 +5611,17 @@ bool CoreChecks::ValidateImageBarrier(const LogObjectList &objects, const Locati
         }
     }
 
+    if (!enabled_features.attachment_feedback_loop_layout_features.attachmentFeedbackLoopLayout) {
+        if (mem_barrier.newLayout == VK_IMAGE_LAYOUT_ATTACHMENT_FEEDBACK_LOOP_OPTIMAL_EXT) {
```

Comment:
nit: I expect layout to not be `VK_IMAGE_LAYOUT_ATTACHMENT_FEEDBACK_LOOP_OPTIMAL_EXT` 99.9% of the time for apps, and I think for other "the feature enables this random enum value" we flip the `if` statements around

```suggestion
    if (mem_barrier.newLayout == VK_IMAGE_LAYOUT_ATTACHMENT_FEEDBACK_LOOP_OPTIMAL_EXT) {
        if (!enabled_features.attachment_feedback_loop_layout_features.attachmentFeedbackLoopLayout) {
```

Reply:
Good point - fixed